checkstyle.xml

```diff
@@ -24,7 +24,7 @@
         <module name="UnusedImports"/>
 
         <module name="ImportControl">
-            <property name="file" value="import-control.xml"/>
+            <property name="file" value="../import-control.xml"/>
```

Comment:
```suggestion
            <property name="file" value="import-control.xml"/>
```
This change breaks the build for me because checkstyle can't find this file. Removing the `../` fixes it.

Reply:
Ok, I see. I was testing the documentation build from within the documentation directory, and this is required to build from there. I will revert this.